### ISSUE TITLE
RuneLite Side Menu/Plugin Menu Scaling Way Too Small

### ISSUE THREAD

**Issue by AcesBullet** [runelite/runelite#18259]
### Type

GPU

### Operating System

Linux

### OS Version

Linux Mint 21.3

### Bug description

Running Linux Mint 21.3 with internal graphics (10th gen i5 vPRO), 1920 x 1080 resolution.  When adjusting the display scaling on the OS from 100% to 125% (or higher) the side menu/plugin menu on RuneLite does not scale up, and in-fact, reduces itself in size.

### Screenshots or videos

![pic1](https://github.com/user-attachments/assets/e42734cc-67a6-409a-a152-5d0465392449)
![pic2](https://github.com/user-attachments/assets/bbea9571-f2dd-49d3-b322-108565f982e5)

### RuneLite version

RuneLite Version: 1.10.37
Launcher Version: 2.7.2

**Comment by cdfisher** [comment]
You can use the "Scale" parameter in the launcher configuration window. Instructions on how to do so can be found [here](https://github.com/runelite/runelite/wiki/RuneLite-Launcher-Configuration).

**Comment by AcesBullet** [comment]
For now I've made a shortcut on my desktop to launch RuneLite through Terminal using the following command:
./RuneLite --scale=2.0
This works well for now, but now I suppose this has turned into a question of why the scaling isn't saved in the RuneLite configure window.

**Comment by capybarista** [comment]
I am having a similar issue on my 1440p monitor - when the desktop is scaled up the RuneLite UI is too small. I can only get it to scale up for integer values (2, 3) which are all too large. Is there a reason we cannot scale by say 1.25 or 1.5?

**Comment by coopermor** [comment]
Non-integer scaling is supported. I believe it is a limitation of your windows manager if it is not. You should look into the settings of your windows manager to see if you can enable it.

**Comment by capybarista** [comment]
I am using Wayland on KDE Plasma and I have it set so that applications can scale on their own - @coopermor do you have any advice on how to further troubleshoot?

**Comment by capybarista** [comment]
Hi @coopermor , I can confirm that non-integer scaling is possible on Wayland, but for whatever reason RuneLite will only scale if my display is set to force all applications to adhere to the system scaling. This means I have to force other applications to scale as well. Any idea how to make the client scale? I am using the Bolt Launcher and I have found that passing the command line args " -J -Dsun.java2d.uiScale=1.5" does not do anything. But if it is set to an integer, it does scale.

**Comment by capybarista** [comment]
Are we sure RuneLite supports integer scaling? I am seeing some differing opinions and have tried various methods that do not seem to work: https://github.com/Adamcake/Bolt/issues/168

**Comment by Caitacular** [comment]
Just wanted to note I'm having the same issue with KDE plasma on CachyOS (Arch Linux derivative). I can set scaling to 2 or 1, but anything in-between gets treated as 1. Since I play at 1440p, this means I have to choose between massively over-scaled or hard to see and click on UI.

**Comment by Caitacular** [comment]
Looks like this is an OpenJDK issue: https://bugs.openjdk.org/browse/JDK-8326428
There's an in-review fix for it so hopefully this'll be resolved once it is accepted and released in a new JRE for runelite to use.

**Comment by CallumDennis** [comment]
Hey just wanted to know if there's been any movement/other solution on this. I'm having exactly the same issue as @Caitacular, same OS and also running a 1440p monitor.
